src/main/java/uk/gov/hmcts/darts/entities/Annotation.java

```diff
@@ -0,0 +1,71 @@
+package uk.gov.hmcts.darts.entities;
```

Comment:
Think these need to be moved to the common package e.g. `uk.gov.hmcts.darts.common.entities`